src/App.js

```diff
@@ -1,16 +1,101 @@
 import React from 'react';
+import TodoList from './components/TodoComponents/TodoList';
+import TodoForm from './components/TodoComponents/TodoForm';
 
 class App extends React.Component {
-  // you will need a place to store your state in this component.
-  // design `App` to be the parent component of your application.
-  // this component is going to take care of state, and any change handlers you need to work with your state
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      todo: []
+    }
+;  }
+
+    addItem = (e) => {
+      let temp = {
+        task: e.target.previousSibling.value,
```

Comment:
This is probably better to be done with `setState` and an item in state that holds the value changing dynamically as you type rather than messing with DOM elements.